pkg/webhooks/sgx/sgx.go

```diff
@@ -36,15 +36,19 @@ type Mutator struct {
 }
 
 const (
-	namespace           = "sgx.intel.com"
-	encl                = namespace + "/enclave"
-	epc                 = namespace + "/epc"
-	provision           = namespace + "/provision"
-	quoteProvAnnotation = namespace + "/quote-provider"
-	aesmdQuoteProvKey   = "aesmd"
+	namespace                = "sgx.intel.com"
+	encl                     = namespace + "/enclave"
+	epc                      = namespace + "/epc"
+	provision                = namespace + "/provision"
+	quoteProvAnnotation      = namespace + "/quote-provider"
+	aesmdQuoteProvKey        = "aesmd"
+	aesmdSocketDirectoryPath = "/var/run/aesmd"
+	aesmdSocketName          = "aesmd-socket"
 )
 
-func getAesmdVolume(needsAesmd bool, epcUserCount int32, aesmdPresent bool) *corev1.Volume {
+func getAesmdVolume(needsAesmd bool, epcUserCount int32, aesmdPresent bool, pod *corev1.Pod) *corev1.Volume {
```

Comment:
Would it be more readable if the function was named `createAesmdVolumeIfNotExists()`?

Reply:
That makes sense, fixed.

Reply:
This function is a constructor effectively. That's why I expected its name to be prefixed with `create` instead of `get`. But that's nitpicky :)

Reply:
Oh, I didn't notice the prefix, I just changed the postfix. Fixed!